### PR TITLE
Replace lxml with defusedxml

### PR DESCRIPTION
**Pull request by metaodi**

### CODE REVIEW

File: ckanext/stadtzhharvest/tests/test_harvester.py [hunk 1]
```diff
@@ -50,3 +50,313 @@ def test_load_metadata_from_path(self):
...
+        eq_(results['count'], num_objects)
+        return results
+
+    def test_harvest_update_dwh(self):
+        data_path = os.path.join(
+            __location__,
+            'fixtures',
+            'DWH'
+        )
+        temp_dir = tempfile.mkdtemp()
```
Comment (stefina) [inline ckanext/stadtzhharvest/tests/test_harvester.py]: In the setup()-method of FunctionalHarvestTest there is already a temp-dir created. No need to create it twice.

File: ckanext/stadtzhharvest/tests/test_harvester.py [hunk 2]
```diff
@@ -50,3 +50,313 @@ def test_load_metadata_from_path(self):
...
+
+            assert result['title'] in expected_titles, "Title does not match result: %s" % result
+
+    def test_harvest_update_geo(self):
+        data_path = os.path.join(
+            __location__,
+            'fixtures',
+            'GEO'
+        )
+        temp_dir = tempfile.mkdtemp()
```
Comment (stefina) [inline ckanext/stadtzhharvest/tests/test_harvester.py]: Similar as above: 
In the setup()-method of FunctionalHarvestTest there is already a temp-dir created. No need to create it twice.

File: ckanext/stadtzhharvest/tests/test_harvester.py (hunk 1 above)
Comment (metaodi) [reply ckanext/stadtzhharvest/tests/test_harvester.py]: done.

File: ckanext/stadtzhharvest/tests/test_harvester.py (hunk 2 above)
Comment (metaodi) [reply ckanext/stadtzhharvest/tests/test_harvester.py]: done.

**Pull request merged**